javatests/arcs/android/entity/DifferentAndroidHandleManagerDifferentStoresTest.kt

```diff
@@ -26,119 +25,106 @@ import kotlin.coroutines.EmptyCoroutineContext
-                )
-            )
+            stores = StoreManager(activationFactory)
         )
         writeHandleManager = EntityHandleManager(
             arcId = "arcId",
             hostId = "hostId",
             time = fakeTime,
             scheduler = schedulerProvider("writer"),
-            stores = StoreManager(
-                activationFactory = ServiceStoreFactory(
-                    app,
-                    fakeLifecycleOwner.lifecycle,
-                    connectionFactory = testConnectionFactory
-                )
-            )
+            stores = StoreManager(activationFactory)
         )
         // Initialize WorkManager for instrumentation tests.
         WorkManagerTestInitHelper.initializeTestWorkManager(app)
-        (fakeLifecycleOwner.lifecycle as LifecycleRegistry).currentState = Lifecycle.State.STARTED
     }
 
     @After
     override fun tearDown() {
         super.tearDown()
-        (fakeLifecycleOwner.lifecycle as LifecycleRegistry).currentState = Lifecycle.State.DESTROYED
+        fakeLifecycleOwner.lifecycleRegistry.handleLifecycleEvent(Lifecycle.Event.ON_DESTROY)
     }
 
-    @Ignore("b/157166918 - Deflake")
     @Test
     override fun collection_referenceLiveness() {
         super.collection_referenceLiveness()
     }
 
-    @Ignore("b/154947352 - Deflake")
     @Test
     override fun collection_removingFromA_isRemovedFromB() {
         super.collection_removingFromA_isRemovedFromB()
     }
 
-    @Ignore("b/154947352 - Deflake")
     @Test
     override fun collection_addingToA_showsUpInQueryOnB() {
         super.collection_addingToA_showsUpInQueryOnB()
     }
 
-    @Ignore("b/154947352 - Deflake")
     @Test
     override fun collection_clearingElementsFromA_clearsThemFromB() {
         super.collection_clearingElementsFromA_clearsThemFromB()
     }
 
-    @Ignore("b/157169321 - Deflake")
     @Test
     override fun singleton_dereferenceEntity_nestedReference() {
         super.singleton_dereferenceEntity_nestedReference()
     }
 
-    @Ignore("b/157171348 - Deflake")
     @Test
     override fun collection_entityDereference() {
         super.collection_entityDereference()
     }
 
-    @Ignore("b/157261807 - Deflake")
     @Test
     override fun singleton_clearOnAClearDataWrittenByB() {
         super.singleton_clearOnAClearDataWrittenByB()
     }
 
-    @Ignore("b/157261828 - Deflake")
     @Test
     override fun collection_dereferenceEntity_nestedReference() {
         super.collection_dereferenceEntity_nestedReference()
     }
 
-    @Ignore("b/157262951 - Deflake")
     @Test
     override fun collection_withTTL() {
         super.collection_withTTL()
     }
 
-    @Ignore("b/157263799 - Deflake")
     @Test
     override fun singleton_referenceLiveness() {
         super.singleton_referenceLiveness()
     }
 
-    @Ignore("b/157262953 - Deflake")
     @Test
     override fun singleton_dereferenceEntity() {
         super.singleton_dereferenceEntity()
     }
```

Comment:
can remove these, even better if you can remove the open modified in the base class.

Reply:
I'd like to keep them open in the base class, since it's valuable to be able to easily put an override in a particular child class to run that test from the IDE. But you're right - i'll delete these. Forgot about that here :)